### PR TITLE
Content update

### PR DESCRIPTION
**Pull request by NeilMcW**
# Description

Updates to 7 day instructions, plus general improvements.

### CODE REVIEW

File: src/main/resources/data/templates/aosinvitation.html [hunk 1]
```diff
@@ -1,270 +1,113 @@
...
-    <head>
-        <style type="text/css">
-            /* Open Sans Light */
-            @font-face {
-                font-family: 'Open Sans';
-                format('woff');
-                font-weight: 300;
-                font-style: normal;
-            }
+{% extends "layout-letter.html" %}
```
Comment (mthava) [inline src/main/resources/data/templates/aosinvitation.html]: I don't think you can do this. PDF service expects a single template file and it has to contain everything in it

File: src/main/resources/data/templates/aosinvitation.html [hunk 2]
```diff
@@ -1,270 +1,113 @@
...
-                font-family: "Open Sans";
-                line-height: 1.15;
-                font-style: normal;
-                margin-top: 5px;
-                margin-bottom: 5px;
-                padding: 0px;
-                text-align: right;
-            }
+        <!-- <img src="/public/images/royal-arms.svg" width="100px;" style="float:left; margin-right: 15px;"> -->
+        <img src="/public/images/HMCTS_BLK_SML_AW.svg" width="200px;" style=" margin-right: 15px; margin-bottom: 30px;">
```
Comment (mthava) [inline src/main/resources/data/templates/aosinvitation.html]: Same here. you can't provide links in this file. All the contents have to be embedded in the file. That is why we were using base64

File: src/main/resources/data/templates/aosinvitation.html [hunk 3]
```diff
@@ -1,270 +1,113 @@
...
...O+rLOpni1E+LQ7x5SN6Ax45lB418HzJwX8zPfAKZZbDa30d0Vy+dqZRqN5qM92PlXiR0TSdIsNPZWjyWTj6I/1Yn6S70T8BkxHBAOwe48QAFJZG0ZyDnFKr77j/8vfX5PNcK4xLAzDBTPT9nSlnO64sntGIFP3hePrj53i65JNPPwwaKRTph86eT8VQiNbGuNmOdtantpdzsoroHfifREbHP4aU70BklOkDyusK+A+nrBoFTuqS1IPr+rLU4TvUW4uNStUBmXKhUbB/tBdLkUbGQasyP0dZf8+j9hHSSH6urENzU9ZN4rJG13vSq2wPQZdwXnTfB9oApP4H+/J2CGMiup+y3ARdobqOUO1v/B68x4pKMC/tdYZHTp5o++p55H8I8uWKSKvR3f7i3Icpi3Mfvonh/Vt47/TIyhBlexr+Xn3qbwfS2QuSf5EbpTw9Shv8DR39WaqrdlT93brPaF84DqTeN8P/Yi69h4zj/pmr1fo3uG46Mkgb3QKv+n0pAz8gIBiAPQBlGy8GoCA/Z023JRLBc5/KYtN0iHFtABrQ/1dBsZ5GeYucGI8WaaupKyhSy8+9JVGn3+6P3Gbz83pKlP9S8tVKzlI70eTAIx4d5pPbI3qZ8ulz8rK0S7NV4QNtANJfP0w+cvzd1FgooFdJ73uxqNH2apOyhlCvSCNf2o5xrJ1Cz4IOOMWWx1fOXtMb5Hc6Rt/Xbf4JVpUOIL7pixZp9am/k33fAyYbggHYPcaZAahPaloV2kyZSQF9Cdbp4xSdYvwbgBb6tDMr5d0R0udhvblrNKqd0cF2SHJ1f1D00Z2VSmVP8tebvUYSeomqDFz679486xqJSXyz9aJeMkAk6ynkH0w91ie/bss/rLZAphYrqazdllNtLMP6Bp6rtZBvykdZ98vxuTTQBiCQnpmP5/ci8lQbqY5l26nIABT0DMyE3F3g07xOfRrupUGU9HnNa32IfHaW/iDPQfmsKSP03fRlfRaWk3zVvxMjO0+qt0i7oNyM/GQ1v6/e5nM5bfN5+FWGwvZH1k10hVXh73Z+bMBExngxAPnh/SllUqcvoEgT2McEPLT7+MuUklbMlTUAt/OkTwm9oDk3XaNSqZ7qkx9TsVuO9jC8NPLkg86TR+0FGVQwLRTzjjvIWFhMCpu6yCDUW7lG06SUkx/dhNznyg3XD4h4lU4OnjFGan+Hf
```
Comment (mthava) [inline src/main/resources/data/templates/aosinvitation.html]: Images have to be embedded like this

**Comment by tchow8** [comment]
Merging, but wait for Neil to sign off after testing in AAT.

**Pull request merged**